numpy/_core/include/numpy/_dtype_api.h

```diff
@@ -162,6 +163,26 @@ typedef NPY_CASTING (resolve_descriptors_function)(
         npy_intp *view_offset);
 
 
+/*
+ * Rarely needed, slightly more powerful version of `resolve_descriptors`.
+ * See also `resolve_descriptors_function` for details on shared arguments.
+ */
```

Comment:
Would be good to add a comment here that this one is private for now given uncertainties about the signature.